main.go

```diff
@@ -1,14 +1,17 @@
 package main // import "github.com/Nitro/sidecar"
 
 import (
+	"context"
 	"os"
 	"os/signal"
 	"runtime/pprof"
 	"time"
 
 	"github.com/Nitro/memberlist"
 	"github.com/Nitro/sidecar/catalog"
+	"github.com/Nitro/sidecar/config"
 	"github.com/Nitro/sidecar/discovery"
+	"github.com/Nitro/sidecar/envoy"
```

Comment:
I think you forgot to commit these files?

Reply:
They are committed. See commit 4ef0166. GH is being an ass for not expanding them :(